server/routerlicious/packages/services-telemetry/src/lumberjack.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "path";
+import { LumberEventName } from "./lumberEventNames";
+import { Lumber } from "./lumber";
+import { LogLevel, LumberType, ITelemetryMetadata, ILumberjackEngine } from "./resources";
+
+// Lumberjack is a telemetry manager class that allows the collection of metrics and logs
+// throughout the service. A list of ILumberjackEngine must be provided to Lumberjack
+// by calling setupEngines() before Lumberjack can be used.
+export class Lumberjack {
+    protected static _instance: Lumberjack | undefined = undefined;
+    private readonly engineList: ILumberjackEngine[] = [];
+    private isSetupCompleted: boolean = false;
```

Comment:
For the sake of consistency, should all `private` variables in Lumber and Lumberjack be prefixed with `_`?